### DISCUSSION TITLE
is deepspeed inference now supporting llama2 replace_with_kernel_inject=True?

### DISCUSSION THREAD

**Discussion by liveforfun** [deepspeedai/DeepSpeed#4290] [Q&A]
I tried to inference with replace_with_kernel_inject=True option with llama2-70b.
but I got some errors.
as far as I know replace_with_kernel_inject=True this option is injecting the high-performance kernels.
but it might be not supporting now right?

**Comment by RezaYazdaniAminabadi** [comment]
Hi @liveforfun 

I am adding the support for this here: https://github.com/microsoft/DeepSpeed/pull/4313
Best,
Reza
